packages/react-percy-webpack/package.json

```diff
@@ -15,16 +15,16 @@
     "lib"
   ],
   "peerDependencies": {
-    "webpack": "^2.2.1"
+    "webpack": "^1.13 || 2.x"
   },
   "dependencies": {
     "interpret": "^1.0.1",
     "memory-fs": "^0.4.1",
-    "virtual-module-webpack-plugin": "^0.1.3",
+    "virtual-module-webpack-plugin": "^0.3.0",
```

Comment:
We'll need to remove this entirely to get `react-percy` working with `create-react-app`, but that's a problem for a separate PR.